frontend/layouts/OnboardingLayout/OnboardingLayout.jsx

```diff
@@ -12,6 +12,12 @@ import styles from './styles';
 
 import { setLoading } from '@/redux/slices/authSlice';
 
+const defaultSteps = [
```

Comment:
I liked your implementation and your thinking beyond putting the onboarding steps globally. It was a good catch.
However, when I update the page, the onboarding steps are lost, since it is not persistent in the database.
<img width="812" alt="Screenshot 2024-09-03 at 12 12 56" src="https://github.com/user-attachments/assets/74e9bf22-6896-43b8-aeec-57d8aec68c89">

I am creating an HOC to make the system redirect the user to the correct onboarding page, so I think it is not necessary to create a slice for the onboarding step.
Please, remove the onboarding steps globally and keep as it as before.
We can discuss it later and make it globally in other task.